src/frontend/komponenter/NesteStegKnapp/NesteStegKnapp.tsx

```diff
@@ -1,29 +1,50 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
 
 import { useNavigate } from 'react-router-dom';
 
 import { Button, HStack } from '@navikt/ds-react';
 
+import { useApp } from '../../context/AppContext';
 import { useBehandling } from '../../context/BehandlingContext';
 import { FanePath } from '../../Sider/Behandling/faner';
+import { Steg } from '../../typer/behandling/steg';
+import { RessursStatus } from '../../typer/ressurs';
+import { Feilmelding } from '../Feil/Feilmelding';
 
 export const NesteStegKnapp: FC<{
     nesteFane: FanePath;
-    children?: React.ReactNode;
+    children: React.ReactNode;
 }> = ({ nesteFane, children }) => {
     const navigate = useNavigate();
+    const { request } = useApp();
 
-    const { behandling } = useBehandling();
+    const { behandling, hentBehandling } = useBehandling();
+    const [feilmelding, settFeilmelding] = useState<string>();
+
+    const gåtTilNesteSteg = () => {
+        settFeilmelding('');
+        if (behandling.steg === Steg.INNGANGSVILKÅR) {
```

Comment:
Håndterer foreløpig bare steget Inngangsvilkår. Må gå opp hvordan det skal løses for stønadsvilkår.